services/transaction_receipt.proto

```diff
@@ -164,4 +164,9 @@ message TransactionReceipt {
      * the newly created NFTs
      */
     repeated int64 serialNumbers = 14;
+
+    /**
+     * In the receipt of a NodeCreate, the id of the newly created node.
```

Comment:
I believe the standard is to populate this even for deletes and updates like we do for other entity types.

Reply:
Yes, deletes and updates will be like other entity types. Here only list created entity ids.

Reply:
I'm saying we don't just list created entity IDs. For all CRUD operations we always populate the ID in the receipt.

Reply:
Yes, our HandleWorkflow takes care of populating the ID in the receipt. When I said "list" here, really should have said: node_id need to be added to this file. The comment is just to match all other entity comment in this file.